src/MongoDB.Driver.Core/TransactionOptions.cs

```diff
@@ -26,6 +27,39 @@ namespace MongoDB.Driver
     /// </summary>
     public class TransactionOptions
     {
+        #region static
+        // public static methods
+        /// <summary>
+        /// Returns a TransactionOptions constructed from the values specified in a BsonDocument.
+        /// </summary>
+        /// <param name="document">The document.</param>
+        /// <returns>A TransactionOptions.</returns>
+        public static TransactionOptions FromBsonDocument(BsonDocument document)
```

Comment:
I'm not sure this should be here, since the only use case is for tests.  Consider moving it to the Test module.

Reply:
I see your point.

We already have a number of ToBsonDocument and FromBsonDocument methods elsewhere.